content/4_Gold/Sliding.mdx

```diff
@@ -141,9 +141,40 @@ For each position of the window, we want to compute some information. For exampl
 </Resources>
 
 ### Implementation
+**Time Complexity: ** $\mathcal{O}(N)$
 
-<IncompleteSection />
+<LanguageSection>
+
+<CPPSection>
 
+```cpp
+vector<int> maxSlidingWindow(vector<int>& nums, int k) {
+        deque<int> d;
+        vector<int> ret;
+        for(int i = 0; i < k; i++){
+            while(!d.empty() && nums[i] > nums[d.back()]){
+                d.pop_back();
+            }
+            d.push_back(i);
+        }
+        for(int i = k; i < nums.size(); i++){
+            ret.push_back(nums[d.front()]);
+            if(!d.empty() && d.front() <= i-k){
+                d.pop_front();
+            }
+            while(!d.empty() && nums[i] >= nums[d.back()]){
+                d.pop_back();
+            }
+            d.push_back(i);
+        }
+        ret.push_back(nums[d.front()]);
+        return ret;
+}
+```
+
+</CPPSection>
+
+</LanguageSection>
```

Comment:
I think there needs to be a newline after this (otherwise the problems heading won't be rendered properly)